shardingsphere-infra/shardingsphere-infra-context/src/main/java/org/apache/shardingsphere/infra/context/metadata/impl/StandardMetaDataContexts.java

```diff
@@ -54,23 +55,8 @@
     private final StateContext stateContext;
     
     public StandardMetaDataContexts() {
-        this(new LinkedHashMap<>(), 
-                new ShardingSphereRuleMetaData(Collections.emptyList(), Collections.emptyList()), null, new ConfigurationProperties(new Properties()));
-    }
-    
-    /**
-     * todo remove this deprecated constructor.
-     * @deprecated old
-     */
-    @Deprecated
-    public StandardMetaDataContexts(final Map<String, ShardingSphereMetaData> metaDataMap, final ShardingSphereRuleMetaData globalRuleMetaData, 
-                                    final ExecutorEngine executorEngine, final ConfigurationProperties props) {
-        this.metaDataMap = new LinkedHashMap<>(metaDataMap);
-        this.globalRuleMetaData = globalRuleMetaData;
-        this.executorEngine = executorEngine;
-        optimizeContextFactory = new OptimizeContextFactory(metaDataMap);
-        this.props = props;
-        stateContext = new StateContext();
+        this(new LinkedHashMap<>(), new ShardingSphereRuleMetaData(Collections.emptyList(), Collections.emptyList()),
+                null, new ConfigurationProperties(new Properties()), new OptimizeContextFactory(new HashMap<>()));
```

Comment:
Is there any reference for this constructor?

Reply:
yes, the ProxyContent is in use.

Reply:
If an empty StandardMetaDataContexts is wanted, we can consider using our new constructor. How do you think?

Reply:
`TableMetaDatas` still needs your attention. Thanks.

Reply:
My oversight. I made the change, but it doesn't seem to have triggered the change.

Reply:
I don't quite understand. `this` calls our new constructor.

Reply:
Got it, therefore this is resolved.

Reply:
yeah!